src/NHibernate/Context/WcfOperationSessionContext.cs

```diff
@@ -50,3 +56,36 @@ public class WcfStateExtension : IExtension<OperationContext>
 		public void Detach(OperationContext owner) { }
 	}
```

Comment:
We cannot, since it implements an interface from `System.ServiceModel`. Not implementing it would be no less binary breaking, since removing an interface is binary breaking. Moreover the interface's members use other `System.ServiceModel` types and could not be left in the class either.

Reply:
In theory we can do this, but I would not want to.